jschart/index.js

```diff
@@ -5374,18 +5379,24 @@ function reset_chart(chart) {
 
   reset_axes_domains(chart);
 
-  chart.chart.svg.attr("height", get_svg_height(chart));
-
+  chart.chart.svg.attr("height", get_svg_height(chart))
+              
   chart.state.reset = true;
 }
 
 function get_svg_height(chart) {
+  console.log(chart.dimensions.viewport_height +
```

Comment:
Is this just some debugging code that got left in?

Reply:
Yes, it was left by mistake. Resolved now. Do you think it will be a good idea to integrate ESlint in JSchart?